__DEFINES/setup.dm

```diff
@@ -788,6 +788,8 @@ SEE_PIXELS	256
 // Special 'weapons', used in damage procs
 #define WPN_HIGH_BODY_TEMP "High Body Temperature"
 #define WPN_LOW_BODY_TEMP  "Low Body Temperature"
+#define INTERNAL "internal"
```

Comment:
I had the odd idea for using it in the future with other things, but that doesn't really make sense thinking about it. Reason they're not pure numbers was just following how the rest of the defines in the stack are done